assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Footer.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Footer component for SearchFunnelWidget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { _x } from '@wordpress/i18n';
```

Comment:
This should be imported in the WordPress dependencies.

Reply:
Thanks, that's done :+1: